### PR TITLE
Adds initial descriptions of all skill levels

### PR DESCRIPTION
**Pull request by pellebjerkestrand**
I'll be going through these and making changes based on the input from our team gathering where we talked about this, but I wanted to make a PR so that it's visible that we are doing this and what we are doing.

Feel free to comment on anything.

### CODE REVIEW

File: levels/level-0.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+Grunnleggende forståelse for HTTP
+
+- Hva er det vanligste verbene og hva brukes de til?
+
+Grunnleggende forståelse for bruk av eksterne APIer over HTTP
+
+- Hva er `fetch` og hvordan brukes det?
+- Hva er CORS og hvorfor finnes det?
+- Hva er en proxy og hvorfor brukes det?
```
Comment (asbjornh) [inline levels/level-0.md:39]: Er dette veldig relevant for utvikling av grensesnitt for episerver-sider?

File: levels/level-4.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Erfaring
+
+8 år eller lengre erfaring i en relevant stilling
+
+## Faglig kompetanse
+
+Veldig god forståelse for med oppsett og tilpasning av verktøy og systemer i bruk i produksjonsprosessen
+
+Veldig god forståelse for HTTP
```
Comment (Reasonable-Solutions) [inline levels/level-4.md:19]: I think perhaps as a sjefsbrukergrensesnittsutvikler all the other things you do are more important than any number of really, really good understanding of http.

File: levels/level-2.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+- Må kunne sette opp en kodebase og en produksjonsprosess slik at en selv og andre kan utvikle
+
+Erfaring med oppsett og tilpasning av verktøy og systemer i bruk i produksjonsprosessen
+
+- Må kunne endre produksjonsprosessen for å tilpasse den endrede/andre behov
+
+God forståelse for HTTP
+
+God forståelse for bruk av eksterne APIer over HTTP
```
Comment (Reasonable-Solutions) [inline levels/level-2.md:31]: HATEOAS!

File: levels/level-1.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+_eller_
+
+Relevant erfaring fra forrige nivå
+
+## Faglig kompetanse
+
+Grunnleggende forståelse for, og erfaring med kildekodehåndteringsverktøy som Git, SVN, Mercurial e.l.
+
+- Må kunne bruke de mest vanlige kommandoene og vite hva de gjør (f.eks. `git [clone, checkout, branch, commit, push, pull, merge]`)
```
Comment (Reasonable-Solutions) [inline levels/level-1.md]: rebase?

File: levels/level-0.md (hunk 1 above)
Comment (Reasonable-Solutions) [reply levels/level-0.md:39]: I see it as a question of fartstid i faget, if you've been around just a bit in the right contexts.

File: levels/level-1.md (hunk 4 above)
Comment (Sriess) [reply levels/level-1.md]: Must be able to only ever use rebase locally

File: levels/level-4.md (hunk 2 above)
Comment (pellebjerkestrand) [reply levels/level-4.md:19]: Yeah, in addition 👌🏻

File: levels/level-1.md (hunk 4 above)
Comment (pellebjerkestrand) [reply levels/level-1.md]: `rebase` at this level? It's not a difficult concept, so maybe 🤔

File: levels/level-0.md (hunk 1 above)
Comment (pellebjerkestrand) [reply levels/level-0.md:39]: And do we really want to tie our skills to Episerver? (The answer is "no").

Comment (Reasonable-Solutions) [reply levels/level-0.md:39]: Perhaps generalize to something about "having some familiarity with "mv**"

File: levels/level-1.md (hunk 4 above)
Comment (Sriess) [reply levels/level-1.md]: It's not among the most commonly used commands, so it's fine just knowing not to use it on any commits that has been pushed to the repository

**Comment by pellebjerkestrand** [comment]
I'll merge this PR and then we can accept other smaller PRs on the content if needed.

**Pull request merged**